papermill_origami/noteable_dagstermill/context.py

```diff
@@ -51,8 +51,9 @@ def op(self) -> Node:
         return self.solid
 
     @property
-    def solid_def(self) -> SolidDefinition:
+    def solid_def(self):
         """SolidDefinition: The current solid definition."""
+        # TODO: this is deprecated in dagstermill; remove later
         return self.solid_def
```

Comment:
@MSeal I'm unsure how we want to refactor this and propagate the changes wherever the `SerializableExecutionContext` is used. I've left it as a TODO for now.